docs/plugins/didcomm.md

```diff
@@ -1,6 +1,8 @@
 ---
 sidebar_position: 3
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # DIDcomm
```

Comment:
```suggestion
# DIDComm
```

"DIDComm" should be the correct ways to write it, saw this also in line 9 and 270, but couldn't make a suggestion in this PR reivew. :D